File: generateclasses/Matiereprixnow_v.java
package entite;
import java.sql.Date;

public class Matiereprixnow_v{
    int idmatiere; 
    double prix; 
    Date dateprix; 

    public Matiereprixnow_v(){ }
    
    public int getIdmatiere(){
        return this.idmatiere;
    }
    public void setIdmatiere(int idmatiere){
        this.idmatiere=idmatiere;
    }
    public double getPrix(){
        return this.prix;
    }
    public void setPrix(double prix){
        this.prix=prix;
    }
    public Date getDateprix(){
        return this.dateprix;
    }
    public void setDateprix(Date dateprix){
        this.dateprix=dateprix;
    }

}

File: generateclasses/Matiereprixnow_v.cs
namespace entite;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("matiereprixnow_v")]
public class Matiereprixnow_v : Motherobj<Matiereprixnow_v>  {
    [Column("idmatiere")]
    public int idmatiere {get; set;} 
    [Column("prix")]
    public double prix {get; set;} 
    [Column("dateprix")]
    public DateTime dateprix {get; set;} 

    public Matiereprixnow_v(){ }
    

}
